clientlibs/js/src/functions/markExperimentPoint.ts

```diff
@@ -12,26 +20,28 @@ export default async function markExperimentPoint(
   status: MARKED_DECISION_POINT_STATUS,
   target?: string
 ): Promise<Interfaces.IMarkExperimentPoint> {
-  try {
-    let data: any = {
-      site,
-      condition,
-      userId,
-      status,
+  let data: markData = {
+    site,
+    condition,
+    userId,
+    status,
+  };
+  if (target) {
+    data = {
+      ...data,
+      target,
     };
-    if (target) {
-      data = {
-        ...data,
-        target,
-      };
-    }
-    const response = await fetchDataService(url, token, clientSessionId, data, Types.REQUEST_TYPES.POST);
-    if (response.status) {
-      return response.data;
-    } else {
-      throw new Error(JSON.stringify(response.message));
-    }
-  } catch (error) {
-    throw new Error(error.message);
+  }
+  const response = await fetchDataService(url, token, clientSessionId, data, Types.REQUEST_TYPES.POST);
+  if (response.status) {
+    return response.data;
+    // return {
```

Comment:
can we just delete commented out "old" code everywhere? If we want to revisit old code, it's always there in git history

Reply:
code removed